src/pages/index.js

```diff
@@ -0,0 +1,207 @@
+import { debounce, sortBy } from "lodash";
+import React, { Component, Fragment } from "react";
+import LazyLoad from "react-lazyload";
+import { Box, Container, Flex, Heading, Text, theme } from "@hackclub/design-system";
+import ClubCard from "components/ClubCard";
+import Header from "components/Header";
+import Footer from "components/Footer";
+import SearchBox from "components/SearchBox";
+import Settings from "components/Settings";
+import axios from "axios";
+import geolib from "geolib";
+import Fuse from "fuse.js";
+import qs from "query-string";
+
+class IndexPage extends Component {
+    constructor(props) {
```

Comment:
Not a big deal, but the pattern of a static for initial state + automatic function binding is preferable:

https://github.com/withspectrum/spectrum/blob/alpha/src/components/chatInput/index.js#L76-L123

Reply:
the majority of this page component should be relocated to its own component/broken up a bit. most of this functionality should be render-able from any page on the site.